src/librustc_middle/mir/interpret/mod.rs

```diff
@@ -398,12 +396,6 @@ impl<'tcx> AllocMap<'tcx> {
     pub fn new() -> Self {
         AllocMap { alloc_map: Default::default(), dedup: Default::default(), next_id: AllocId(0) }
     }
-
-    /// Obtains a new allocation ID that can be referenced but does not
-    /// yet have an allocation backing it.
-    ///
-    /// Make sure to call `set_alloc_id_memory` or `set_alloc_id_same_memory` before returning such
-    /// an `AllocId` from a query.
     pub fn reserve(&mut self) -> AllocId {
```

Comment:
Can this become private now? Otherwise, why did you remove the doc-comment?

Reply:
oh right. can be private, I moved the doc comment to a function on `TyCtxt`. This function is still on `AllocMap`.